CMakeLists.txt

```diff
@@ -86,6 +92,11 @@ set(SYMFORCE_EIGEN_TARGET "Eigen3::Eigen" CACHE STRING
 # Setup
 # ==============================================================================
 
+# If we're cross-compiling macOS arm64 wheels, set the architecture appropriately
+if("$ENV{CIBW_ARCHS_MACOS}" STREQUAL "arm64")
+  set(CMAKE_OSX_ARCHITECTURES arm64)
+endif()
```

Comment:
at some point it might make sense to split out toolchain files

Reply:
Hmm possibly yeah - I don't think we'd want to be defining toolchains for users to use, it'd just be for our CI (if a user wants a toolchain they should define it, I don't _think_ it's correct for a library to define a toolchain?).  Not sure how that'd look for this wheel building process (i.e. how we'd pipe it through)